Sofa/framework/Config/CMakeLists.txt

```diff
@@ -23,19 +23,20 @@ define_property(TARGET
 # it. (E.g. some versions of Visual Studio have 'solution folders')
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-### Sofa using type double or float or both
-set(SOFA_FLOATING_POINT_TYPE both CACHE STRING
+### Sofa using type double or float
+set(SOFA_FLOATING_POINT_TYPE double CACHE STRING
     "Type used for floating point values in SOFA. It actually determines:
     - what template instanciations will be compiled (via the definition of the
     SOFA_FLOAT and SOFA_DOUBLE macros)
-    - what is the type behind the 'SReal' typedef used throughout SOFA. (If 'both'
-    is selected, SReal defaults to double.)")
-set_property(CACHE SOFA_FLOATING_POINT_TYPE PROPERTY STRINGS float double both)
+    - what is the type behind the 'SReal' typedef used throughout SOFA.")
+set_property(CACHE SOFA_FLOATING_POINT_TYPE PROPERTY STRINGS float double)
```

Comment:
If we want to be complete, there is a third floating-point type: `long double`

See https://en.cppreference.com/w/cpp/language/types

Reply:
Funny, I looked at it just before https://en.wikipedia.org/wiki/Quadruple-precision_floating-point_format and even half-float 16b as well.
Thing is, long double for MSVC is actually just  `double`...

Reply:
interesting